### ISSUE TITLE
Unable to add-owner 

### ISSUE THREAD

**Issue by caraballoj** [CocoaPods/CocoaPods#5196]
* [x ] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?
pod trunk add-owner ManagedStatsOC bob@bytefly.com

## What did you expected to happen?
I expected to successfully add a new owner to the pod.

## What happened instead?
Instead I received the following message "[!] No owner found with the specified email address."

## Podfile

> ℹ Please paste the contents of your Podfile below.    

Pod::Spec.new do |s|
  s.name             = 'ManagedStatsOC'
  s.version          = '0.1.21'
  s.summary          = 'A pod for building Apps by MangagedApps.co.'
  s.homepage         = 'https://github.com/managedapps/ManagedStatsOC'
  s.license          = 'MIT'
  s.author           = { 'Bob Pascazio' => 'bob@bytefly.com' }
  s.source           = { :git => 'https://github.com/managedapps/ManagedStatsOC.git', :tag => s.version.to_s }
  s.platform         = :ios, '7.0'
  s.requires_arc     = true

  s.source_files       = 'Pod/Classes/*.{h,m}'
  s.resources          = 'Pod/Classes/DisclaimerVC.xib'
# s.resource_bundles = {
#    'ManagedStatsOC' => 'Pod/Classes/*.xib'
#  }

# s.public_header_files = 'Pod/Classes/**/*.h'
# s.frameworks       = 'UIKit', 'MapKit'
  
end

```ruby
# Paste your Podfile contents HERE.
```

**Comment by orta** [comment]
Has someone registered that account? You can't create a new account via 'add-owner'

On my phone, replies will be curt.

**Comment by caraballoj** [comment]
Yes I have. I'm even using it in two projects.

**Comment by caraballoj** [comment]
@orta will you be able to help me with this? I tried again today with no success.

**Comment by orta** [comment]
What is the email?

**Comment by segiddins** [comment]
`bob@bytefly.com` isnt a registered owner on trunk

**Comment by caraballoj** [comment]
@segiddins How do I make bob@bytefly.com a registered owner on trunk if not by using 'pod trunk add-owner ManagedStatsOC bob@bytefly.com'

**Comment by orta** [comment]
Here's the guide - https://guides.cocoapods.org/making/getting-setup-with-trunk.html

**Comment by caraballoj** [comment]
@orta yes I used this guide. It states, 

Adding other people as contributors

The first person to push a Podspec version to Trunk can add other maintainers. For example, to add kyle@cocoapods.org to the library ARAnalytics:

$ pod trunk add-owner ARAnalytics kyle@cocoapods.org

which is what I did, but was unsuccessful in adding a contributor. 

Any help in this matter would be greatly appreciated.
